.github/workflows/nightly.yml

```diff
@@ -0,0 +1,29 @@
+name: Generate and publish report nightly
+on:
+  schedule:
```

Comment:
why not add this cron scheduler to the previous job ?

Reply:
Good catch, thank you :)
I merged nightly.yml and merge.yml together and renamed it to `report.yml` to better reflect the contents. I amended the previous commit.